vignettes/loo2-large-data.Rmd

```diff
@@ -17,12 +17,15 @@ params:
 
 This vignette demonstrates how to do leave-one-out cross-validation for large
 data using the __loo__ package and Stan. There are two approaches covered:
-LOO with subsampling and LOO using approximatations to posterior distributions. 
+LOO with subsampling and LOO using approximations to posterior distributions. 
 Some sections from this vignette are excerpted from the papers
 
-* Magnusson, M., Riis Andersen, M., Jonasson, J. and Vehtari, A. (2019). Leave-One-Out Cross-Validation for Model Comparison in Large Data.
+* Magnusson, M., Riis Andersen, M., Jonasson, J. and Vehtari, A. (2020). Leave-One-Out Cross-Validation for Model Comparison in Large Data.
+Proceedings of the 23rd International Conference on Artificial Intelligence and Statistics (AISTATS), in PMLR 108. [arXiv preprint arXiv:2001.00980](https://arxiv.org/abs/2001.00980).
```

Comment:
Is PMLR number known already?

Reply:
Yes. It is part of the AISTATs 2020 template.